src/Particle.js

```diff
@@ -305,15 +305,29 @@ class Particle {
 	 * Add a device to a product or move device out of quarantine.
 	 * @param  {Object} options Options for this API call
 	 * @param  {String} options.deviceId Device ID
+	 * @param  {Object} options.file    Path to or Buffer of a single-column list of device IDs, device serial numbers, device IMEIs or device ICCIDs
+	 *                                  Either a path or Buffer of the file contents in Node, or a File or Blob in the browser.
```

Comment:
I had a hard time reading these docs with the type information at the beginning and having the second set of type info expand beyond the first?

```
A file that contains a single-column list of device IDs, device serial numbers, device IMEIs, or devie ICCIDs.
Either a path or Buffer of the file contents in Node, or a File or Blob in the browser.
```

Maybe, or flip it around so it is type info then description of content?

Reply:
took a crack at it over here 👉 https://github.com/particle-iot/particle-api-js/pull/109/commits/16cca544c06074b3e8f030bdc05f7e7af2481670

if you spot other issues, please add them to this story 👉 https://app.clubhouse.io/particle/story/45778/improve-jsdoc-data and we can tackle 'em in a follow-on PR 👍